src/plugins/login/loginPlugin.ts

```diff
@@ -28,10 +28,11 @@ export class AzureLoginPlugin {
     try {
       if (subscriptionId && clientId && secret && tenantId) {
         authResult = await loginWithServicePrincipalSecretWithAuthResponse(clientId, secret, tenantId);
-      } else {
-        await open('https://microsoft.com/devicelogin');
-        authResult = await interactiveLoginWithAuthResponse();
-      }
+      } 
```

Comment:
Removed `open` for now just because of the invalid typings file

Reply:
actually - if you install latest typescript it works.